app/screens/draft_scheduled_post_options/send_draft.test.tsx

```diff
@@ -0,0 +1,171 @@
+        const props = {
+            channelId: 'channel_id',
+            channelName: 'channel_name',
+            rootId: '',
+            channelType: General.OPEN_CHANNEL,
+            bottomSheetId: Screens.DRAFT_SCHEDULED_POST_OPTIONS,
+            currentUserId: 'current_user_id',
+            maxMessageLength: 4000,
+            useChannelMentions: true,
+            userIsOutOfOffice: false,
+            customEmojis: [],
+            value: 'value',
+            files: [],
+            postPriority: '' as unknown as PostPriority,
+            persistentNotificationInterval: 0,
+            persistentNotificationMaxRecipients: 0,
+            draftReceiverUserName: undefined,
+        };
+        const wrapper = renderWithIntlAndTheme(
+            <SendDraft
+                {...props}
+            />,
+        );
+        const {getByTestId} = wrapper;
+        fireEvent.press(getByTestId('send_draft_button'));
+        expect(dismissBottomSheet).toHaveBeenCalledTimes(1);
+    });
+
+    it('should render the component for scheduled post', () => {
+        const props = {
+            channelId: 'channel_id',
+            channelName: 'channel_name',
+            rootId: '',
+            channelType: General.OPEN_CHANNEL,
+            bottomSheetId: Screens.DRAFT_SCHEDULED_POST_OPTIONS,
+            currentUserId: 'current_user_id',
+            maxMessageLength: 4000,
+            useChannelMentions: true,
+            userIsOutOfOffice: false,
+            customEmojis: [],
+            value: 'value',
+            files: [],
+            postPriority: '' as unknown as PostPriority,
+            persistentNotificationInterval: 0,
+            persistentNotificationMaxRecipients: 0,
+            draftReceiverUserName: undefined,
+            draftType: DRAFT_TYPE_SCHEDULED as DraftType,
+        };
+        const wrapper = renderWithIntlAndTheme(
+            <SendDraft
+                {...props}
+            />,
+        );
+        const {getByText} = wrapper;
+        expect(getByText('Send')).toBeTruthy();
+    });
+
+    it('should call removeDraft when clearing a draft', () => {
+        const mockHandleSendMessage = jest.fn();
+        jest.mock('@hooks/handle_send_message', () => ({
+            useHandleSendMessage: () => ({
+                handleSendMessage: mockHandleSendMessage,
+            }),
+        }));
+
+        const TestClearDraft = () => {
+            const serverUrl = useServerUrl();
+            const channelId = 'channel_id';
+            const rootId = 'root_id';
+            const draftType = DRAFT_TYPE_DRAFT;
+
+            React.useEffect(() => {
+                // This simulates the clearDraft function logic
+                if (draftType === DRAFT_TYPE_DRAFT) {
+                    removeDraft(serverUrl, channelId, rootId);
+                }
+            }, []);
+
+            return null;
+        };
+
+        renderWithIntlAndTheme(<TestClearDraft/>);
```

Comment:
You are creating a new component, rendering that component, and checking if the component you have created is calling the function. This is not testing anything.